rubicon_ml/viz/base.py

```diff
@@ -79,6 +84,19 @@ def serve(self, in_background=False, dash_kwargs={}, run_server_kwargs={}):
             the 'port' argument can be provided here to serve the app on a
             specific port.
         """
+        if in_background:
+            warnings.warn(
+                "The `in_background` argument is deprecated and will have no effect, "
+                "Background processing is now handled by `jupyter_mode`."
+            )
```

Comment:
Should add DeprecationWarning as the warning type here - (e.g. `warnings.warn(message, DeprecationWarning)`